CHANGELOG.md

```
@@ -1,3 +1,9 @@
+## [1.11.0]
+
+#### New Features
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)